src/sources/file/line_agg.rs

```diff
@@ -293,6 +293,7 @@ fn add_next_line(buffered: &mut BytesMut, line: Bytes) {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use pretty_assertions::assert_eq;
```

Comment:
Interesting, I'll have to remember that one for my own tests. Too badd we can't turn that on across all the sources at once.

Reply:
I noticed it being used at the `file/mod.rs` and it looked really good.

Can't we import it at `/lib.rs` with an old `extern crate pretty_assertions` + macro use syntax?